torch/csrc/jit/passes/onnx/function_extraction.cpp

```diff
@@ -1045,6 +1118,59 @@ NodeAttrNameMap ONNXFunctionExtraction(
   return fe.run();
 }
 
+Node* ONNXGetPreviousScope(std::shared_ptr<Graph>& graph) {
+  auto* last_node = graph->nodes().back()->prev();
+  auto* scope_node = GetNodeOfMostRecentScope(last_node);
+  auto* attr_node = scope_attr_graph_->create(prim::TracedModuleForward);
+  attr_node->setScope(scope_node->scope());
+  TORCH_INTERNAL_ASSERT(
```

Comment:
Can we log something about `scope_node` here so the error message is useful in case this fails?